samples/nfc/tag_reader/prj.conf

```diff
@@ -9,6 +9,11 @@ CONFIG_SPI_0=y
 CONFIG_SPI=y
 CONFIG_SPI_0_NRF_SPI=y
 
+#Disable unused SPI instance
+CONFIG_SPI_1=n
```

Comment:
This instance is used on nRF5340. It seems that UART0 (used for logging) & SPI0 share HW resources and they cannot be used simultaneously on nRF5340.

Reply:
nRF5340 has a different dedicated `.conf` file. This changes does not have any impact on it

Reply:
yes, you are right